guides/micronaut-data-r2dbc-repository/micronaut-data-r2dbc-repository.adoc

```diff
@@ -0,0 +1,151 @@
+common:header.adoc[]
+
+The application exposes some REST endpoints and stores data in a MySQL database using Micronaut Data R2DBC.
```

Comment:
should include a brief summary of what R2DBC is, how it's different, why they'd use it, etc.

Reply:
I added:

> What is [R2DBC](https://r2dbc.io/)?
> 
> The Reactive Relational Database Connectivity (R2DBC) project brings reactive programming APIs to relational databases.